glesys.go

```diff
@@ -231,7 +253,12 @@ func (d *Driver) GetSSHPort() (int, error) {
 
 // GetSSHUsername returns username for use with ssh
 func (d *Driver) GetSSHUsername() string {
-	return "root"
+	if d.Platform == "KVM" {
```

Comment:
Wouldn't it be good to be able to configure this? 
Use kvm as fallback or something.